### ISSUE TITLE
Include spatial data manipulation

### ISSUE THREAD

**Issue by psolymos** [borealbirds/tutorials#1]
We are supposed to move towards the 'intersect stuff on the fly' approach, so this would be also necessary. @dstralberg and @psolymos has code/material.

**Comment by NKSBarker** [comment]
Agreed. @crosbya1  was prompted by Erin to work on this as well - tracking down layers to generate covariates for offsets. 
I will work on linking derived variables back to intersections back to layers.
